src/devenv/cli.py

```diff
@@ -0,0 +1,445 @@
+import os
```

Comment:
does it make sense to split this file into multiple files? like every command own file?

Reply:
I think for now it's best to keep it in one file for faster iterations.

Later on, once things stabilize, we can split it up.